worker/raft/raftclusterer/worker.go

```diff
@@ -158,6 +158,10 @@ func (w *Worker) updateConfiguration(
 }
 
 func (w *Worker) configOps(configuredServers, newServers map[raft.ServerID]raft.ServerAddress) []configOp {
+	if len(newServers) == 0 {
+		logger.Warningf("peergrouper reported 0 API server addresses; not removing all servers from cluster")
```

Comment:
Is it really a warning? There isn't anything the user can do. Perhaps Info would be better?

Reply:
Yeah, fair call - I'll demote it.